multilingual-news-app-using-sveltekit-with-directus/index.md

```diff
@@ -0,0 +1,443 @@
+author:
+  name: 'Ekekenta Clinton'
+  avatar_file_name: './ekekenta-clinton.png'
+---
+
+## Introduction
+Svelte is a new JavaScript framework for building user interfaces for web applications. It offer a better way for handling UI updates when compared with React, Angular, or Vue.js. In this tutorial, you'll learn how to build a multilingual news application using Sveltekit with Directus i18n API.
+
+## Before You Start
+You will need:
+- [Node.js v20.11.1](https://nodejs.org/) or later
+- A code editor on your computer.
+- Create and login to your [Directus Cloud account](https://directus.cloud/) and have install the Directus SDK `npm install @directus/sdk`.
+- Some knowledge of React and Svelte.
+
+The code for this tutorial is available on my [GitHub repository](https://github.com/icode247/directus-i18n-app); feel free to clone it to follow along.
+
+## The Concept of i18n
+
+i18n is the process of creating applications that support multiple languages. The phrase i18n represents the first and the last letters of the word **internationalization**. The number 18 denotes the number of letters between the first and the last letter. The main goal of i18n is to conveniently create applications that support multiple languages or locations without having to redesign the application to add those languages.
+
+
+## Understanding Directus i18n API
+
+Directus is an open-source headless Content Management System that provides a robust API for managing content. It offers an i18n API that enables the internationalization and localization of content within the system.
+
+The Directus i18n API allows you to manage translations for various content types, such as articles, products, or any other data stored in the Directus system.
+
+With the Directus i18n API, you can build multilingual applications or websites that serve localized content to users based on their preferred language or region. 
+
+## Setting up a new project.
+
+Let's start by setting up a new Svelte project using [degit](https://github.com/Rich-Harris/degit), by running the command below:
+
+```
+ npx degit sveltejs/template directus-i18n-app
+```
+
+The above command will scalfold a new Svelte project with the required dependencies. 
+
+Next, change directory into the project folder and install the dependencies.
+
+```
+cd directus-i18n-app && npm install
+```
+
+Then install the **svelte-routing** package as a dev dependency:
+```
+npm i -D svelte-routing
+```
```

Comment:
````suggestion
Let's start by setting up a new Svelte project using the [degit scaffolding tool](https://github.com/Rich-Harris/degit):

```
npx degit sveltejs/template directus-i18n-app
```

Install the required dependencies:

```
cd directus-i18n-app 
npm install
npm install @directus/sdk
npm install --save-dev svelte-routing
```

In the `src/libs` directory, create a `directus.js` file to create and export a Directus SDK instance:

```
import { createDirectus, rest } from '@directus/sdk';
const directus = createDirectus('<YOUR DIRECTUS URL>').with(rest());
export default directus;
```

Be sure to provide your real project URL.
````

This should be a continuation of 'Before You Start'. As you can see, we've brought all of the dependency installation into a single instruction to make this more concise. The SDK install has also been moved here as it's a related step.

I've also added the boring boilerplate of setting up the SDK to this section.